### PR TITLE
Reverify coverage

### PR DESCRIPTION
**Pull request by alexflav23**

### CODE REVIEW

File: app/uk/gov/hmrc/digitalservicestax/controllers/package.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ package object controllers {
     def escapeHtml: String = HtmlFormat.escape(in).toString
   }
 
-  implicit class OrderedYesNo(l: List[String]) {
+  implicit class OrderedYesNo(val l: List[String]) extends AnyVal {
```
Comment (adamdye-hmrc) [inline app/uk/gov/hmrc/digitalservicestax/controllers/package.scala:26]: Why is this needed?

Comment (alexflav23) [reply app/uk/gov/hmrc/digitalservicestax/controllers/package.scala:26]: https://docs.scala-lang.org/overviews/core/value-classes.html#summary_of_limitations

Comment (alexflav23) [reply app/uk/gov/hmrc/digitalservicestax/controllers/package.scala:26]: It's a performance optimisation, it's not strictly needed, but it makes perfect sense to have it in this type of scenario.

Comment (adamdye-hmrc) [reply app/uk/gov/hmrc/digitalservicestax/controllers/package.scala:26]: Oh awesome, lgtm then.
